templates/repo/issue/fields/dropdown.tmpl

```diff
@@ -3,7 +3,8 @@
 	{{/* FIXME: required validation */}}
 	<div class="ui fluid selection dropdown {{if .item.Attributes.multiple}}multiple clearable{{end}}">
 		<input type="hidden" name="form-field-{{.item.ID}}" value="0">
-		<i class="dropdown icon"></i>
+		{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+		{{svg "octicon-x" 14 "remove icon"}}
```

Comment:
And what happened here?

Reply:
Fomantic automatically inserts `<i class="remove icon">` via JS when at least 1 item is present and the icon is absent in HTML. With this present, fomantic will no longer try to add it and use this icon instead.

Reply:
Fomantic JS here:

https://github.com/go-gitea/gitea/blob/fe2fe852634c0f2ee4f7360fc424f2675e198cdf/web_src/fomantic/build/semantic.js#L3180-L3186